x-pack/plugins/cloud_security_posture/public/pages/findings/findings.test.tsx

```diff
@@ -22,6 +22,13 @@ import type { DataView } from '../../../../../../src/plugins/data/common';
 
 jest.mock('../../common/api/use_kubebeat_data_view');
 jest.mock('../../common/api/use_cis_kubernetes_integration');
+jest.mock('../../common/hooks/use_theme', () => ({
```

Comment:
Instead of mocking this I suggest adding a theme provider to our `<TestProvider />`

Reply:
we could this if you prefer, but we don't own the provider, we get it from eui for all kibana (not in our app providers) so adding it seems more complicated than mocking it.

Reply:
Not sure what you mean, we do add it explicitly, Kibana does not do that for us (we add it [here](https://github.com/elastic/kibana/blob/0a4f0fc5046943564cba3ba9952ede180fa352ae/x-pack/plugins/cloud_security_posture/public/application/index.tsx#L21))

Reply:
removed